src/passes/DeadArgumentElimination.cpp

```diff
@@ -72,9 +72,11 @@ struct DAEFunctionInfo {
   // see inhibits our optimizations, but TODO: an export
   // could be worked around by exporting a thunk that
   // adds the parameter.
+  // This is atomic so that we can write to it from any function at any time
+  // during the parallel analysis phase which is run in DAEScanner.
   std::atomic<bool> hasUnseenCalls;
 
-  DAEFunctionInfo() { hasUnseenCalls.store(false); }
+  DAEFunctionInfo() { hasUnseenCalls = false; }
```

Comment:
Can this be initialized up with the declaration so this whole constructor can be deleted?

```
std::atomic<bool> hasUnseenCalls = false;
```